docs/semgrep-code/semgrep-pro-engine-examples.md

```diff
@@ -233,44 +221,38 @@ export const PLANCK_CONSTANT = 6.62606896e-34;
 export const EMPLOYEE_TABLE_NAME = "Employees";
 ```
 
-Semgrep Pro Engine matches the first call without any change to the rule.
+Semgrep matches the first call without any change to the rule.
 
-:::tip Try it out
-Run Semgrep Pro Engine in the cloned [Semgrep Pro Engine testing repository](https://github.com/semgrep/semgrep-pro-tests). Go to `docs/constant_propagation_dangerous_calls` and run the following command:
+To test this in the cloned [Semgrep testing repository](https://github.com/semgrep/semgrep-pro-tests),go to `docs/constant_propagation_dangerous_calls` and run the following command:
 
 ```sh
 semgrep --config pro.yaml . --pro
 ```
-:::
 
 ### Propagating values
 
-In the previous example, we only cared whether the string was constant or not, so we used `”...”`, but constant propagation also propagates the constant value. To illustrate the use of Semgrep Pro Engine with constant propagation, the rule from the previous section is changed to search for calls to `dangerous("Employees");`.
+In the previous example, it only mattered cared whether the string was constant or not, so the example used `”...”`, but constant propagation also propagates the constant value. To illustrate the use of Semgrep with constant propagation, the rule from the previous section is changed to search for calls to `dangerous("Employees");`.
 
 #### Java
 
 <iframe title="Semgrep example no prints" src="https://semgrep.dev/embed/editor?snippet=YqyD" width="100%" height="432" frameborder="0"></iframe>
 
-With Semgrep Pro Engine, this rule matches the last three calls to `dangerous`, since these calls are selected from the `Employees` table, though each one obtains the table name differently:
+With Semgrep, this rule matches the last three calls to `dangerous`, since these calls are selected from the `Employees` table, though each one obtains the table name differently:
 
-:::tip Try it out
-Run Semgrep Pro Engine in the cloned [Semgrep Pro Engine testing repository](https://github.com/semgrep/semgrep-pro-tests). Go to `docs/constant_propagation_propagating_values` and run the following command:
+To test this in the cloned [Semgrep testing repository](https://github.com/semgrep/semgrep-pro-tests), go to `docs/constant_propagation_propagating_values` and run the following command:
 
 ```sh
 semgrep --config pro.yaml . --pro
 ```
-:::
 
 #### JavaScript and TypeScript
 
 <iframe title="Semgrep example no prints" src="https://semgrep.dev/embed/editor?snippet=pORk" width="100%" height="432" frameborder="0"></iframe>
 
-With Semgrep Pro Engine, this rule matches the last three calls to `dangerous`, since these calls are selected from the `Employees` table, though each one obtains the table name differently:
+With Semgrep, this rule matches the last three calls to `dangerous`, since these calls are selected from the `Employees` table, though each one obtains the table name differently:
```

Comment:
```suggestion
With Semgrep, this rule matches the last three calls to `dangerous`, since these calls are selected from the `Employees` table, though each one obtains the table name differently.
```